.github/workflows/build-wheel-wrapper.yml

```diff
@@ -21,5 +21,7 @@ on:
 jobs:
   python-wrapper-wheel:
     name: Python Wrapper Wheel
-    uses: ecmwf-actions/reusable-workflows/.github/workflows/python-wrapper-wheel.yml@main
```

Comment:
@iainrussell is this change from `ecmwf-actions/reusable-workflows` to `ecmwf/reusable-workflows` now for all our actions to be recommended?